pkgs/by-name/ec/ec2-instance-selector/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "ec2-instance-selector";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "amazon-${pname}";
+    tag = "v${version}";
+    hash = "sha256-4J66/LiFFeUW20du2clqjz9ozLV+Sn2VVqF9VISXpb0=";
+  };
+
+  vendorHash = "sha256-ocysHrbkmFQ96dEVJvc5YuuBiaXToAcMUUPFiLpMCpU=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.versionID=${version}"
+    "-X=github.com/aws/amazon-ec2-instance-selector/v3/pkg/selector.versionID=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/${pname}
+  '';
+
+  meta = {
+    description = "A CLI tool and go library which recommends instance types based on resource criteria like vcpus and memory";
+    homepage = "https://github.com/aws/amazon-ec2-instance-selector";
+    changelog = "https://github.com/aws/amazon-ec2-instance-selector/tags/${src.rev}";
```

https://github.com/NixOS/nixpkgs/issues/367739
```suggestion
    changelog = "https://github.com/aws/amazon-ec2-instance-selector/tags/v${finalAttrs.version}";
```